.env.example

```diff
@@ -63,3 +63,8 @@ CLIENT_SECRET_GITHUB_LOGIN=
 
 CLIENT_ID_GITLAB_LOGIN=
 CLIENT_SECRET_GITLAB_LOGIN=
+
+CAPTCHA_SECRET=
+CAPTCHA_ENABLED=
+
+NEXT_PUBLIC_CAPTCHA_SITE_KEY=
```

Comment:
Oh just realized, this will need to be replaced after build right? If so,  we'll need to make use of this https://github.com/Infisical/infisical/blob/main/frontend/scripts/replace-standalone-build-variable.sh which is being called from here https://github.com/Infisical/infisical/blob/main/Dockerfile.standalone-infisical#L136

Reply:
will add!